packages/fixie/tsconfig.json

```diff
@@ -9,10 +9,10 @@
     "moduleResolution": "node16",
     "jsx": "react",
     "esModuleInterop": true,
-    "outDir": ".",
+    "outDir": "dist",
```

Comment:
I tried very hard to do this a couple weeks ago and failed because of CJS/ESM/Typescript `moduleResolution` nightmares (the resulting package couldn't be consumed in a straightforward manner in NextJS projects).

We should definitely do this, but I think we need to do it in concert with other work.